src/app/song/song-search/song-search.component.html

```diff
@@ -0,0 +1,57 @@
+<mat-card>
+  <form (submit)="onSearch(searchForm)" #searchForm="ngForm">
+
+    <mat-form-field>
+      <input matInput name="songName" ngModel>
+    </mat-form-field>
+
+    <mat-form-field>
+      <input matInput name="artistName" ngModel>
+    </mat-form-field>
+
+    <mat-form-field>
+      <input matInput name="genreName" ngModel> <!-- TODO : make this a checkbox with the genre enum -->
+    </mat-form-field>
+
+    <button mat-raised-button type="submit"></button>
+
+  </form>
+</mat-card>
```

Comment:
Maybe add some names to these input boxes?

Reply:
let me finish the functionalities of my tasks first

Reply:
styling is not necessary now

Reply:
Add the names, is that so much to ask?